### ISSUE TITLE
Signaling for use of consumables

### ISSUE THREAD

**Issue by TiberiusB** [valnet/valuenetwork#112]
Members of the network invest by purchasing consumables. Other members use these consumables in different projects. Access to consumables can be restricted by their scope (in what projects they can be used). The "investor" might want to maintain a certain control over how the consumables are used. Therefore, we need a signaling system to announce the "investor" whenever someone else plans to use a consumable. The investor might have the power to approve of disprove the use, if the consumable has a restricted scope of use. 
In other words, members always chose where and how to allocate their resources, based on their goals and risk tolerance. The "investor" needs to be in control of the assets he/she invest. 
Some consumables can have free access, no permission required. 
See more on the wiki - Resource type
http://valuenetwork.referata.com/wiki/Resource_type
